client/utils/contentProfiles.ts

```diff
@@ -81,10 +82,57 @@ export function getUnusedProfileFields(
     profile: IEditorProfile,
     includeGroupCheck: boolean = true
 ): Array<IProfileFieldEntry> {
+    // Temporary FIXME: Will be removed once we add support for CVs as fields in coverage profiles
+    if (profile.name === 'coverage') {
+        const fieldsFromProfile = getProfileFields(profile);
+
+        return orderBy(
+            fieldsFromProfile
+                .filter((field) => (includeGroupCheck && field.field.group == null) || !field.field.enabled),
+            'name'
+        );
+    }
+
+    const cvs = planningApi.vocabularies.getCustomVocabularies();
+    const cvIds = cvs.map((x) => x._id);
+    const cvFieldEntries = cvs
+        .map((x, i) => ({
+            field: {
+                enabled: false,
+                group: undefined,
+                index: i,
+            },
+            name: x._id,
+            schema: {
+                type: 'custom_vocabulary',
+                required: false,
+            }
+        }) as IProfileFieldEntry);
+
+    const fieldsFromProfile = getProfileFields(profile);
+    const usedCVs = fieldsFromProfile.filter(
+        (fieldEntry) =>
+            fieldEntry.schema?.type === 'custom_vocabulary'
+            && cvIds.includes(fieldEntry.name)
+            && fieldEntry.field.enabled
+    );
+
+    const unusedCVs = cvFieldEntries.filter((field) => {
+        const fieldSchema = field.schema;
+
+        return fieldSchema?.type === 'custom_vocabulary' && !usedCVs.some((usedFieldEntry) =>
+            usedFieldEntry.name === field.name);
+    });
+
     return orderBy(
-        getProfileFields(profile).filter(
-            (item) => (includeGroupCheck && item.field.group == null) || !item.field.enabled
-        ),
+        fieldsFromProfile
+            .filter((field) => (includeGroupCheck && field.field.group == null) || !field.field.enabled)
+            .filter((field) => {
+                const fieldSchema = field.schema;
```

Comment:
Variable is only used once and naming is not any better than value. Why would you need it?

It would be good to also use a single `filter`